e2e/create/create_test.go

```diff
@@ -30,26 +29,18 @@ import (
 	"go.uber.org/zap/zaptest"
 )
 
-// TODO parallel seems to be buggy.  Not certain why, but we need to figure out if running with the operator
-// deployed in the cluster helps
-// We may have a threadsafe problem where one test starts messing with another test
-var parallel = *flag.Bool("parallel", false, "run tests in parallel")
-
-var validImage = "cockroachdb/cockroach:v21.1.7"
-var nonExistentImage = "cockroachdb/cockroach-non-existent:v21.1.999"
-var invalidImage = "nginx:latest"
-var crdbVersion = "v21.1.7"
-var relatedImageEnvName = "RELATED_IMAGE_COCKROACH_v21_1_7"
+var (
```

Comment:
worth plopping these into e2e.go as well? or meh

Reply:
I left these here since they weren't used anywhere else. I'm on the meh side 😄